Demo/Demo/SwiftUIComponents/SwiftUIPaymentButtonDemo.swift

```diff
@@ -26,7 +26,7 @@ struct SwiftUIPaymentButtonDemo: View {
         ZStack {
             VStack {
                 Picker("Funding Source", selection: $fundingIndex) {
-                    ForEach(fundingSources.indices) { index in
+                    ForEach(fundingSources.indices, id: \.self) { index in
```

Comment:
What is `\.self` needed for?

Reply:
@scannillo we get a `Non-constant range: not an integer range` warning in swift lint